### PR TITLE
Adds Infogram shortcode to our bakery

### PR DESCRIPTION
**Pull request by davisshaver**
See https://github.com/fusioneng/shortcake-bakery/issues/11

### CODE REVIEW

**Comment by davisshaver** [comment]
Build is failing due to PHPCS. Here's the report I get for `class-infogram.php`:
```
vagrant@vip:/srv/www/wp-content/themes/vip/fusion-theme/lib/shortcake-bakery$ !-1
phpcs inc/shortcodes/class-infogram.php

FILE: ...ion-theme/lib/shortcake-bakery/inc/shortcodes/class-infogram.php
----------------------------------------------------------------------
FOUND 6 ERRORS AND 3 WARNINGS AFFECTING 9 LINES
----------------------------------------------------------------------
  2 | ERROR   | Missing file doc comment
  5 | ERROR   | Missing class doc comment
 13 | ERROR   | Public method name "Infogram::get_shortcode_ui_args"
    |         | is not in camel caps format
 32 | ERROR   | Missing parameter comment
 33 | ERROR   | Tag cannot be grouped with parameter tags in a doc
    |         | comment
 37 | WARNING | Line exceeds 85 characters; contains 117 characters
 45 | WARNING | Line exceeds 85 characters; contains 112 characters
 47 | WARNING | Line exceeds 85 characters; contains 101 characters
 57 | ERROR   | Tag cannot be grouped with parameter tags in a doc
    |         | comment
----------------------------------------------------------------------

Time: 82ms; Memory: 2.5Mb
```
Are we enforcing strict PHPCS standards (including the doc rules)?

**Comment by goldenapples** [comment]
@davisshaver Running `phpcs` from the command line like that with no arguments uses the default (Zend?) standard, which is very different from the WordPress core conventions we follow. So it'll definitely fail like that. Try running `grunt phpcs`, or `phpcs --standard=WordPress`?

I'm not totally sure why Travis isn't loading the WordPress ruleset, it should be.

Can you update with a phpcs ruleset file like we use in our other plugins ([here's the file](https://github.com/fusioneng/image-shortcake/blob/master/phpcs.ruleset.xml) and it gets called from Gruntfile like [this](https://github.com/fusioneng/image-shortcake/blob/master/Gruntfile.js#L35)), and see if the build passes?)

**Comment by danielbachhuber** [comment]
> Can you update with a phpcs ruleset file like we use in our other plugins (here's the file and it gets called from Gruntfile like this), and see if the build passes?)

This should fix it.

**Comment by davisshaver** [comment]
@danielbachhuber could you review? Also need the logo file for Infogram.

**Comment by danielbachhuber** [comment]
> could you review?

Looks pretty good

> Also need the logo file for Infogram.

https://www.dropbox.com/s/wj0rtruab44yvba/infogram.svg?dl=0

**Comment by davisshaver** [comment]
@danielbachhuber review/merge?

File: assets/images/svg/icon-infogram.svg [hunk 1]
```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (danielbachhuber) [inline assets/images/svg/icon-infogram.svg]: Good place for it :+1:

**Pull request merged**